sdk/storage/azure-storage-blob/tests/test_common_blob.py

```diff
@@ -1176,7 +1176,13 @@ def test_soft_delete_blob_without_snapshots(self, **kwargs):
 
         try:
             self._setup(storage_account_name, storage_account_key)
+            before_props = self.bsc.get_service_properties()
+            assert before_props['delete_retention_policy'].days is None
+            assert before_props['delete_retention_policy'].enabled is False
```

Comment:
Do you think we actually want to have these checks in place _before_ enabling? My bet is that this won't always be disabled at the beginning since we don't sleep when disabling soft delete. If it doesn't fully disable the test will fail and I'm not sure if we care about that or not?